pkg/api/types.go

```diff
@@ -221,6 +224,24 @@ type GCEPersistentDiskVolumeSource struct {
 	ReadOnly bool `json:"readOnly,omitempty"`
 }
 
+// A ISCSI Disk can only be mounted as read/write once.
+type ISCSIDiskVolumeSource struct {
+	// Required: iSCSI target IP
+	TargetIP string `json:"targetIP,omitempty"`
```

Comment:
*target portal* looks a better accepted term. name and description are now updated